src/v/utils/log_hist.h

```diff
@@ -241,3 +250,11 @@ using log_hist_internal = log_hist<std::chrono::microseconds, 26, 8ul>;
  * the first bucket to greater than 91 days in the last bucket.
  */
 using log_hist_read_dist = log_hist<std::chrono::minutes, 16, 4ul>;
+
+/*
+ * This histogram has units of milliseconds instead of microseconds, and is
+ * used for measuring the Kafka client quota delays on the scale of less than
+ * 1 milliseconds in the first bucket to greater than 32 seconds in the last
```

Comment:
Ahh. This is fine until somebody changes `max_kafka_throttle_delay_ms` to >30s. Not sure what to do about that.

Reply:
If the delays reach >30s, we no longer care about the distribution I don't think. 30s vs 5 minutes doesn't make much of a difference for debugging. At that point, the sum of the delays is more useful.